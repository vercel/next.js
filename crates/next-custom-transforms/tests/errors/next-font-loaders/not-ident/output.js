const { a  } = Inter({
    variant: '400'
});
const [b] = Inter({
    variant: '400'
});
const { e  } = {};
