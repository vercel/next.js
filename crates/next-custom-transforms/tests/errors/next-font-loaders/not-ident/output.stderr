  x Font loader calls must be assigned to an identifier
   ,-[input.js:3:1]
 2 | 
 3 | const { a } = Inter({
   :       ^^^^^
 4 |   variant: '400',
   `----
  x Font loader calls must be assigned to an identifier
   ,-[input.js:7:1]
 6 | 
 7 | const [b] = Inter({
   :       ^^^
 8 |   variant: '400',
   `----
